docroot/themes/custom/uids_base/templates/uids/banner.html.twig

```diff
@@ -90,7 +89,7 @@
                   {{ link.link_text }}
                   <i role="presentation" class="fas fa-arrow-right"></i>
                 </a>
-              {% else %}
+              {% elseif link_element == 'title' %}
                 {# Display as a pseudo-link. #}
                 <div aria-hidden="true" {{ button_attributes.addclass(link.link_classes) }} {% if banner_aria_describedby %} id="{{ banner_aria_describedby }}" {% endif %}>
                   {{ link.link_text }}
```

Comment:
does this just need to be set to banner_title?

Reply:
wait, that just reprints the banner title as the button text. i don't think that is the desired result